README.md

```diff
@@ -8,15 +8,15 @@ Tokens will be validated for:
 * Expiration
 * Signature
 * Valid issuer
-* Valid Client/Audience
+* Valid Audience
 
 Basic usage:
 
 ``` java
 // 1. build the parser
 JwtVerifier jwtVerifier = new JwtHelper()
                             .setIssuerUrl("https://dev-123456.oktapreview.com/oauth2/ausar5cbq5TRooicu812")
-                            .setClientOrAudience("my-audience")
+                            .setAudience("my-audience")
```

Comment:
Do you think we should provide examples for an `accessToken` and `idToken`? This might help clear up the `setAudience()` method.

nit: I know `setX` is common in most languages, but does is it more clear to change this to `for`? Ex: `forAudience()` and `forIssuer()`.